### DISCUSSION TITLE
Flows exported and then imported in other Nodered - Impossible to attach Smartnora

### DISCUSSION THREAD

**Discussion by JN-CLS** [andrei-tatar/node-red-contrib-smartnora#256] [Q&A]
Hi, 
I had 2 Nodered projects running on different servers.
I wanted to integrate my nodered with smartnora into the other one.
Both already had config nodes from different services.
I just exported all flows from the smartnora project and imported into the other one.
Now the smartnora config is disabled and email field is greyed out without info and no password entry field displayed.
It thinks that it is already configured, but of-course the credentials were not taken over I believe.
Anyone an idea how to re-enable the configuration node to set it up in the other nodered project ?

**Comment by andrei-tatar** [comment]
@JN-CLS I will have a look. It might be an oversight. Probably easiest for now is to delete the config node and create a new one.

Isn't the login button enabled?
![image](https://user-images.githubusercontent.com/6473183/230003666-6a0f4ed6-3c8e-4517-8a00-d2e28e4cb601.png)

**Comment by JN-CLS** [comment]
Ok I deleted all flows and nodes, uninstalled Nora and reinstalled via palette manager.Then I took 1 device and connected with config node and that worked.Imported all flows again.Now all seems to work again.Will try to put it on GitHub this evening after work.Thx for all support.From PhoneMet vriendelijke groeten, Most kind Greetings, Sincères salutations, Mit freundlichen Grüßen,JanOp 5 apr. 2023 08:52 schreef Andrei Tatar ***@***.***>:
@JN-CLS I will have a look. It might be an oversight. Probably easiest for now is to delete the config node and create a new one.
